Cleared three lines of white space in the middle of the program

Taranw opened the pull request:
Simplification error that didn't conform to PEP 8 standards.

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards.

emmet0r reviewed (changes requested):
You modified the wrong file - t01_final_story.py instead of t04_refactored. Take another detailed look at the instructions and let us know if you have questions!

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok. Returned t01 to it's original state

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok. Returned t01 to it's original state

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok. Returned t01 to it's original state

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok. Returned t01 to it's original state

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok. Returned t01 to it's original state

Taranw committed:
Merge branches 'Taran_and_Karmadri' and 'master' of https://github.com/spring-2019-csc-226/t04-master into Taran_and_Karmadri

Taranw committed:
Cleared three lines of white space in the middle of the program
Simplification error that didn't conform to PEP 8 standards. moved the corrected code to the refactored file. Added the name of editors and google doc link. Spoke about PEP 8 error around "dead" and confirmed it was A-ok. Returned t01 to it's original state.lowercase included
